provisioner/ansible/provisioner.go

```diff
@@ -76,6 +79,9 @@ type Provisioner struct {
 	ansibleVersion    string
 	ansibleMajVersion uint
 	generatedData     map[string]interface{}
+
+	setupAdapterFunc   func(ui packer.Ui, comm packer.Communicator) (error, string)
```

Comment:
I would've expected the signature to be `func(ui packer.Ui, comm packer.Communicator) (string, error)`. Is there a reason why the return is `(error, string)` and not `(string, error)`?

Reply:
you're right, I should fix this.